packages/web-components/src/components/va-radio/test/va-radio.e2e.ts

```diff
@@ -488,7 +488,7 @@ describe('va-radio', () => {
 
     const element = await page.find('va-radio >>> .usa-label--required');
     expect(element).toEqualHtml(`
-      <span class="usa-label--required" part="required">
```

Comment:
Does part="required" need to be removed? I think teams use this attribute to style the element in some situations (:part pierces the shadow dom).